Week1/homework/select-4.js

```diff
@@ -0,0 +1,32 @@
+const mysql = require('mysql');
+const con = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'newWorld'
+});
+
+con.connect();
+
+const select_query = [
+  "select name from country where population>=8000000", // 4/i
+  "select name from country where name LIKE '%lands'", //4/ii
```

Comment:
correct but would be better if we do `%lands%` to find anything that has lands

Reply:
I have corrected it in step-5 file.